lib/src/query_parameters/keys_query.dart

```diff
@@ -4,15 +4,20 @@ class KeysQuery extends Queryable {
   final int? offset;
   final int? limit;
 
-  KeysQuery({this.limit, this.offset});
+  const KeysQuery({
+    this.limit,
+    this.offset,
+  });
 
-  Map<String, dynamic> buildMap() {
-    return {'offset': this.offset, 'limit': this.limit};
+  Map<String, Object?> buildMap() {
+    return {
```

Comment:
because of the comma (`,`) at the end makes dart formatter append a new line
see https://docs.flutter.dev/development/tools/formatting#using-trailing-commas